### DISCUSSION TITLE
Issue with astropy version>=6.0 and gammapy v1.1

### DISCUSSION THREAD

**Discussion by MicaelAndrade** [gammapy/gammapy#5230] [Help] [answered]
**Gammapy version**
v1.1

**Bug description**
I was trying to make a new gammapy installation and found this problem happening:
`No module named 'astropy.coordinates.angle_utilities`
When using gammapy code that import astropy's angle_utilities functions.

**To Reproduce**
I believe just updating astropy to version 6.0.0 would make this error happen.

**Other information**
I believe this happened because of the [new astropy version 6.0.0](https://docs.astropy.org/en/v6.0.0/changelog.html#) from two weeks ago: 
"Removed deprecated angle parsing and formatting utilities from angle_utilities. Use the functions from angle_formats instead."
Edit: I temporarily fixed this by reverting to Astropy version 5.3.4. However, when installing Gammapy, the latest version of Astropy (6.0.0) is currently being installed, which may pose a problem for new users.

[a comment by an automated account is not shown]

**Comment by registerrier** [comment]
Thanks @MicaelAndrade . Indeed, one has to impose astropy<6.0 for gammapy 1.0.1 and 1.1. 

The recently released 1.0.2 is compatible with astropy v6.0 and so will be v1.2. For now, we don't plan to release a v1.1.1

**Comment by lhaerer** [comment]
A similar issue seems to exist with pydantic. I had to impose pydantic==1.4 (current version is 2.5.3).

**Comment by registerrier** [comment, marked as answer]
This is solved with v1.0.2 and 1.2. With v1.1 be careful to impose astropy<6.0 and pydantic<2.0.

Moving this to discussions, to keep track of the possible installation issue.
